README.md

```diff
@@ -0,0 +1,6 @@
+### LaunchDarkly Sample Javascript Application  ###
+We've built a simple console application that demonstrates how LaunchDarkly's SDK works.  Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/).
+##### Build instructions  #####
+1. Drop the snippet from the [Tutorial](https://app.launchdarkly.com/quickstart#/tutorial) section into the `<head>` section of `index.html`
+2. Copy your feature flag key from your LaunchDarkly dashboard into the snippet in `index.html` 
```

Comment:
This should not be referred to as a snippet. Just "into `index.html`" should be fine.